### PR TITLE
Absolute and relative path passed as argument return matching fields

### PR DESCRIPTION
**Pull request by adexandros**
### Fixed Issues (if relevant)

1. urlResolver support for absolute paths #79

### Manual testing scenarios

1. Passed absolute and relative paths for CMS, Product and Category pages
2. Got always same response with the additional absolute_path field

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/UrlRewriteGraphQl/Model/Resolver/UrlRewrite.php [hunk 1]
```diff
@@ -74,17 +74,20 @@ public function resolve(
         };
         
         if (isset($args['url'])) {
-            $url = $args['url'];
-            if (substr($url, 0, 1) === '/' && $url !== '/') {
-                $url = ltrim($url, '/');
+            $parsedUrl = parse_url($args['url']);
+            $url = $parsedUrl['path'];
+            if ($parsedUrl['path'][0] === '/' && $parsedUrl['path'] !== '/') {
```
Comment (paliarush) [inline app/code/Magento/UrlRewriteGraphQl/Model/Resolver/UrlRewrite.php:79]: `$url` variable can be used here as before, no need to change lines 78-79 (from the original revision).

**Review by paliarush — CHANGES_REQUESTED** [review]
@adexandros Thanks for the PR.

1. Please cover changes with web API functional tests (at least positive)
2. This PR will conflict with https://github.com/magento/graphql-ce/pull/94 , please consider consolidating them. Then we will avoid merge conflicts and will be able to deliver them faster.

**Comment by naydav** [comment]
Hi @adexandros,
I would like to ask you to finish this PR
Of course, if that doesn't bother you
Thanks

**Comment by naydav** [comment]
Will be merge in scope of https://github.com/magento/graphql-ce/pull/208

**Pull request closed without merging**